### ISSUE TITLE
Number of messages archived is not reported in the Server Event

### ISSUE THREAD

**Issue by rbeckman-nextgen** [nextgenhealthcare/connect#3475]
Imported Issue. Original Details:
Jira Issue Key: MIRTH-3580
Reporter: wayneh
Created: 2015-01-20T18:01:03.000-0800

**Comment by rbeckman-nextgen** [comment]
The event log should now show the correct number of messages archived. 

Note for testers: Make sure you also test when not all pruned messages will have a message archived. For instance if you are exporting Transformed content and some of your messages do not have transformed content, then your messages pruned and messages archived should be different.

Imported Comment. Original Details:
Author: wayneh
Created: 2015-01-21T14:16:38.000-0800

**Comment by rbeckman-nextgen** [comment]
Verified that the Data Pruner Events now show up in the Server's Event Browser. I also verified the stipulation that archived messages may not match the number of, say, transformed messages if exporting this content type. I simply made a JS Reader channel to return an arbitrary string each 1ms. After letting this run for a little bit, I modified the channel to include a blank transformer step. After this ran for some time, I ran the data pruner to archive the destination transformed. Visually verified the numbers coincided with the numbers expected after each channel stop.

Imported Comment. Original Details:
Author: jaysenp
Created: 2015-01-22T15:58:20.000-0800
